docs/installation.md

```diff
@@ -86,10 +86,8 @@ Note that the `.env` file should **not** be exposed online as it contains the en
 ##### Application Key
 
 The `.env` file contains the setting `APP_KEY` which holds your application key.
-Use the `php artisan key:generate` command to set the application key to a random value.
-If you don't want to use this command and choose to set the key yourself, it should be 32 characters long.
-
-**If the application key is not set, your user sessions and other encrypted data will not be secure!**
+Normally, Composer sets this key for you while installing the dependencies.
+If it is missing, Lychee will not work. Then, you can use the `php artisan key:generate` command to set the application key to a random value.
```

Comment:
```suggestion
If the key is not set, Lychee will not work. Then, you can use the `php artisan key:generate` command to set the application key to a random value.
```

If one reads sloppily or in a hurry, one could think that "it" refers to "Composer" and I was wondering at first why Composer should be missing.

(Full Sheldon Cooper Mode: on)
And strictly speaking, the key is always there (so it cannot be missing), it can only have an empty value.
(Full Sheldon Cooper Mode: off)